src/scripts/components/Chart.jsx

```diff
@@ -1,66 +1,104 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import ReactHtmlParser from 'react-html-parser';
 import { Bar, Doughnut, Line } from 'react-chartjs-2';
 
 import { Icon } from '.';
 import Tooltip from './Tooltip';
 
-const renderChart = (opts) => {
-  switch (opts.type.toLowerCase()) {
-    case 'bar':
-      return (<Bar {...opts} />);
-    case 'doughnut':
-      return (<Doughnut {...opts} />);
-    case 'line':
-      return (<Line {...opts} />);
-    default:
-      return (<Bar {...opts} />);
-  }
-};
+class Chart extends React.Component {
+  state = {
+    isChartData: false,
+  };
 
-const renderNoData = () => (
-  <div className="chart__without-data">
-      Sorry! There&apos;s nothing to show.
-    <p>
-      Once data is available for this section, it will appear here.
-    </p>
-  </div>
-);
+  legendCallback = function(chart) {  // eslint-disable-line
+    const { labels } = chart.data;
```

Comment:
Add `datasets` here so you can use it below instead of `chart.data.datasets`.

Reply:
done